collection_requirements.rst

```diff
@@ -0,0 +1,145 @@
+Example: `meta/runtime.yml <https://github.com/ansible-collections/collection_template/blob/main/meta/runtime.yml>`_
+* MUST define the minimim version of Ansible which this collection works with
+  * If the Collection works with Ansible 2.9, then this should be set to `>=2.9.10`
+
+Modules & Plugins
+------------------
+
+Documentation
+~~~~~~~~~~~~~~
+All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
+
+* Use the FQCN for ``M(...)`` See `Linking within module documentation<https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
+* Use collection version numbers for `version_added` (otherwise `version_added_collection` must be provided, but this is discouraged)
+
+All module and plugin ``EXAMPLES`` MUST:
+
+* Use FQCN for module (or plugin) name.
+* for modules (or plugins) left in ansible-base use ``ansible.builtin.template``
+
+Other items:
+* You MUST Use the FQCN for ``extends_documentation_fragment:``, unless you are referring to doc_fragments from ansible-base
+
+
+Contributor Workflow
+====================
+
+Changelogs
+~~~~~~~~~~
+
+To give a consistent feel for changelogs across collections, and ensure for collections included in the ``ansible`` package we suggest you use `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_
+
+
+Preferred (in descending order):
+
+1. Use antsibull-changelog (preffered)
+2. Provide ``changelogs/changelog.yaml`` in the `correct format <https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md>`_
+3. Provide a link to the changelog file (self-hosted) (not recommended)
+
+Please note that the porting guide is complied from `changelogs/changelog.yaml` (sections `breaking_changes`, `major_changes`, `deprecated_features`, `removed_features`). So if you use option 3, you will not be able to add something to the porting guide.
+
+Versioning and deprecation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+FIXME Details here
+* Deprecation vs Collection version
+* How many Collection major versions to give
+
+Repo management
+===============
+
+Repo name
+~~~~~~~~~
+
+For collections under ansible-collections the repo SHOULD be named ``NAMESPACE.COLLECTION``
+
+Branch name and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All new repos under `ansible-collections<https://github.com/ansible-collections>`_ MUST have `main`` as the default branch.
+
+Existing repos SHOULD be converted to use ``main``
+
+Repo Protections:
+* Allow merge commits: disallowed
+
+Branch protections MUST be enforced
+* Require linear history
+* Include administrators
+
+CI Testing
+===========
+
+At a minimim ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ``devel`` branch so that you find out about new linting requirements earlier.
```

Comment:
```suggestion
At a minimum ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier.
```